jsonb-spring-adapter/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1,2 @@
+io.avaje.jsonb.spring.JsonBAutoConfiguration
```

Comment:
Is it more consistent to be named JsonbAutoConfiguration ?  
(more consistent with JsonbHttpConverterAutoConfiguration that is)

Reply:
I'm bad at naming things as previously stated

Reply:
oh wait yeah the b should be lowercase

Reply:
yeah sorry, I should have been explicit, I was only referring to the b being lowercase